extensions/src/test/java/org/pentaho/platform/plugin/services/metadata/PentahoAsyncThreadRunnerTest.java

```diff
@@ -0,0 +1,82 @@
+/*!
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * Foundation.
+ *
```

Comment:
[#7 professionalize code for asynchronous populate cache](https://jira.pentaho.com/browse/BACKLOG-36132)